plugins/pulp_rpm/plugins/migrations/0011_new_importer.py

```diff
@@ -16,7 +15,7 @@
 
 
 def migrate(*args, **kwargs):
-    for type_id in (RPM.TYPE, SRPM.TYPE):
+    for type_id in ('rpm', 'srpm'):
```

Comment:
Better to use `RPM._content_type_id` and `SRPM._content_type_id` instead of strings.